doc/nrf/ug_matter_device_certification.rst

```diff
@@ -243,6 +243,14 @@ Matter dependent certification inheritance
 If your product uses qualified Bluetooth stack and certified Thread libraries provided as part of the |NCS|, you can *inherit* certification from Nordic Semiconductor, provided that you do not introduce any changes to these stacks.
 In practice, this means reusing Nordic Semiconductor's certification identifiers, which were obtained as a result of the official certification procedures.
 
+.. note::
+   The inheritance is granted by the related SDO after you join the organization and apply for certification by inheritance.
```

Comment:
certification by inheritance is an application specific for the Thread Group
the Wi-Fi SDO has a QuickTrack or Derative certification option,
the Bluetooth QDID can be obtained by Bluetooth Qualification Process with
“No Required Testing”.